master/pkg/model/task.go

```diff
@@ -45,6 +45,8 @@ const (
 	TaskTypeTensorboard TaskType = "TENSORBOARD"
 	// TaskTypeCheckpointGC is the "CHECKPOINT_GC" job type for the enum public.job_type in Postgres.
 	TaskTypeCheckpointGC TaskType = "CHECKPOINT_GC"
+	// GlobalAccessScopeID
+	GlobalAccessScopeID AccessScopeID = 0
```

Comment:
it seems... odd that we're adding this here.  Surely this was a special number before, right?  Like, `0` already meant this and we're not adding new meaning to it, right?

Reply:
This had been were AccessScopeID had been defined, it was odd that we didn't already have a value defined for this before, 0 already corresponded to global access scope (because the workspace scope associated with the permission would come in from the DB as an empty value)